### PR TITLE
Mds 2399 contact verification - add an arrow

### PR DESCRIPTION
**Pull request by nayr974**
image

### CODE REVIEW

**Review by mhuseinov — APPROVED** [review]
:fire:

File: services/core-web/src/components/Forms/noticeOfWork/EditNoWContacts.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ const renderContacts = ({
         {fields
           .map((field, index) => (
             // eslint-disable-next-line react/no-array-index-key
-            <Col lg={12} sm={24} key={index}>
+            <Col lg={12} sm={24} key={`${index}_${fields.length}`}>
```
Comment (mmumby) [inline services/core-web/src/components/Forms/noticeOfWork/EditNoWContacts.js:67]: can you revert these changes? This is what caused the party_guids fields to clear when adding a new contact

**Review by mmumby — APPROVED** [review]
